avabot.py

```diff
@@ -189,8 +198,14 @@ def parse_slack_output(slack_rtm_output):
     output_list = slack_rtm_output
     if output_list and len(output_list) > 0:
         for output in output_list:
-            if output['type'] == 'message' and 'text' in output:
+            if output['type'] == 'message' and 'text' in output:                
                 text = output['text']
+                if output['user'] != BOT_ID: # prevent sending Ava into a loop
+                    for keyword in MAGIC_BS:
```

Comment:
Use `if any(keyword in text) for keyword in MAGIC_BS` prevent multiple messages and reduce an indent level?